docs/setup/custom-config.mdx

```diff
@@ -234,3 +267,118 @@ If this parameter is left empty (`[]`), users can upload files from any URL with
+
+```bash
+{
+    "permanent_storage": {
+        "location": "local",
+        "bucket": "s3_bucket_name"
+    },
+    "paths": {
+        "root": "/home/mindsdb/var",
+        "content": "/home/mindsdb/var/content",
+        "storage": "/home/mindsdb/var/storage",
+        "static": "/home/mindsdb/var/static",
+        "tmp": "/home/mindsdb/var/tmp",
+        "cache": "/home/mindsdb/var/cache",
+        "locks": "/home/mindsdb/var/locks"
+    },
+    "auth":{
+        "http_auth_enabled": true,
+        "http_permanent_session_lifetime": 86400,
+        "username": "username",
+        "password": "password"
+    },
+    "gui": {
+        "autoupdate": true
+    },
+    "api": {
+        "http": {
+            "host": "127.0.0.1",
+            "port": "47334",
+            "restart_on_failure": true,
+            "max_restart_count": 1,
+            "max_restart_interval_seconds": 60
+        },
+        "mysql": {
+            "host": "127.0.0.1",
+            "port": "47335",
+            "database": "mindsdb",
+            "ssl": true,
+            "restart_on_failure": true,
+            "max_restart_count": 1,
+            "max_restart_interval_seconds": 60
+        },
+        "mongodb": {
+            "host": "127.0.0.1",
+            "port": "47336",
+            "database": "mindsdb"
+        }
+    },
+    "cache": {
+        "type": "local",
+        "connection": "redis://localhost:6379"
+    },
+    "logging": {
+        "handlers": {
+            "console": {
+                "enabled": true,
+                "level": "INFO"
+            },
+            "file": {
+                "enabled": false,
+                "level": "INFO",
+                "filename": "app.log",
+                "maxBytes": 524288,
+                "backupCount": 3
+            }
+        }
+    },
+    "ml_task_queue": {
+        "type": "local",
+        "host": "localhost",
+        "port": 6379,
+        "db": 0,
+        "username": "username",
+        "password": "password"
+    },
+    "file_upload_domains": [],
+    "web_crawling_allowed_sites": []
+}
+```
+
+Next, start MindsDB providing this `config.json` file.
+
+```bash
+python -m mindsdb --config=/path-to-the-extended-config-file/config.json
+```
+</Accordion>
+
+</AccordionGroup>
+
+## Modifying Config Values
+
+Users can modify config values by directly editing the `config.json` file.
+
+Users can edit the config values through an HTTP request or MindsDB Editor. In this case, the `config.auto.json` file is created to store the updated `config.json` file. The `config.auto.json` file is stored in the location defined in the `paths.root` config parameter.
+
+<Tip>
+Modifying config values through MindsDB Editor, provided that `"http_auth_enabled": true`, can be done in Settings.
+
+Modifying config values through an HTTP request:
+
+```bash
+curl --request PUT \
+  --url http://127.0.0.1:47334/api/config \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "key": "value"
```

Comment:
Therefore, here should be 
```js
{
    "auth": {
        "key": "value"
    }
}
```
May be delete this block? I don''t think that anyone want to modify config via HTTP request. It is mostly for GUI.